css/main.css

```diff
@@ -232,81 +225,98 @@ footer {
     bottom: 0;
 }
 
-footer a p {
+footer i p {
     display: inline;
 }
 
-
-#myBtn {
-    display: none;
-    position: fixed;
-    bottom: 20px;
-    right: 30px;
-    z-index: 99;
-    font-size: 36px;
-    border: none;
-    outline: none;
-    background-color: rgba(95, 95, 95, 0.74);
-    color: white;
-    cursor: pointer;
-    padding: 12px 25px;
-    border-radius: 20%;
-  }
-  
-  #myBtn:hover {
-    background-color: #555;
-  }
-
-@media (max-width: 730px) {
-    .menu li {
-        width: 100%;
-        justify-content: center;
-        max-height: 80px;
-        overflow: hidden;
-    }
+footer i {
+    padding-right: 10px;
 }
 
-@media (max-width: 729px) {
+
+@media (max-width: 760px) {
     .menu {
-        height: 20px;
         width: 100%;
         display: flex;
         justify-content: flex-end;
-        align-content: center;
-        padding: 10px;
+        align-content: stretch;
         box-sizing: border-box;
-        transform: translateY(100%);
+        top: 0;
     }
 
-    .nav-trigger {
-        border: 0;
+    #nav-trigger {
+        box-sizing: border-box;
+        border-style: none;
         display: flex;
         justify-content: flex-end;
+        color: #fff;
+        padding: 10px 10px 5px 10px;
+        margin-right: 10%;
+        border-bottom: 3px solid transparent;
+        text-transform: uppercase;
+        font-weight: 400;
+        font-family: 'Raleway', sans-serif;
+        font-size: 20px;
+        height: 100%;
+        background: #333;
+        outline: none;
     }
 
-    .menu {
+    #nav-trigger:hover {
+        border-bottom: 3px solid;
+        border-bottom-color: rgb(230, 45, 45);
+        transition: border-bottom-color 0.5s;
+    }
+
+    .site-menu {
         display: flex;
+        box-sizing: border-box;
+        position: absolute;
+        top: 71px;
+        right: 0;
+        height: 320px;
+        width: 200px;
+        background-color: #333;
+        justify-content: center;
+        transform: translateX(100%);
     }
 
-    /* .menu > li{
-        max-height: 10%;
-        justify-content: right;
-        transform: translateY(10%);
-    } */
+    .site-menu ul {
+        padding: 20px;
+        list-style-type: none;
+        margin: 0;
+        display: flex;
+        flex-wrap: wrap;
+        align-content: flex-start;
+    }
 
-    .menu > li > a:hover {
-        transform: translateY(100%);
+    .site-menu li {
+        margin-top: 15px;
+        color: #fff;
+        flex-basis: 100%;
         display: flex;
-        flex-direction: column;
-        transition: 0.5s;
     }
-}
 
+    .site-menu a {
+        display: flex;
+        padding: 10px;
+        box-sizing: border-box;
+        text-decoration: none;
+        align-items: center;
+        height: 100%;
+        width: 100%;
+    }
+
+    #nav-trigger:focus ~ .site-menu, .site-menu:hover {
```

Comment:
Czy powinien to być `focus` czy jak używjąc taba najadę na ten element to mi się otworzy menu?

Reply:
Przy klasie 'focus' element pozostaje aktywny po kliknięciu - wg mnie łatwiejsze w dalszej obsłudze niż 'hover', dlatego wybrałam. Zmieniam na 'hover'.